docs/vscode-extension/envchecker-help.md

```diff
@@ -149,6 +149,13 @@ The configurations in *.fx/configs/config.local.json* are like:
 
 Note that the `botEndpoint` should use https protocol.
 
+## Sideloading Permission Disabled
+Your Microsoft 365 tenant admin hasn't enabled sideloading permission for your account. You can't install your app to Teams! 
```

Comment:
Sideloading Permission is a setting of your Microsoft 365 tenant. It is required to be enblaed before you can install any customized Teams App.